### ISSUE TITLE
Rename Touchable to Pressable?

### ISSUE THREAD

**Issue by ekryski** [lelandrichardson/react-primitives#76]
This might be just semantics but since Touchable implements `onPress` and various of "press" methods, maybe it makes more sense for it to be called `Pressable`?

**Comment by MoOx** [comment]
Not sold on this.
Touchable could have onTouchStart, onTouchEnd etc. onPress is like "onTouch" but without thinking about start/end. Imo it's better to keep it this way.

**Comment by ekryski** [comment]
Yeah, I'm kind of on the fence on this as well. It seems like a bit of a "make work" project purely for semantics but I did hear @lelandrichardson mention it on a recent podcast so thought I would kick up an issue.

**Comment by jcampbell05** [comment]
Should be intractable if anything :) or button
